### ISSUE TITLE
UUIDs

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#4856]
From Slack (sigh)

> How long do the UUIDs (Temporary universal unique identifier for use only in linking records in the specimen bulkloader to records in other bulkloaders.) stick around? Do we have to manually delete them off of records?

I believe they are permanent and must be manually deleted, but you can use the [other ID bulk-unloader](https://arctos.database.museum/tools/BulkDeleteOtherId.cfm).

**Comment by dustymc** [comment]
I unfortunately don't have the ability to know when that thumb drive with downloaded UUID-linked data has been discovered in the back of the drawer and loaded, so I treat them like all other identifiers.

**Comment by sharpphyl** [comment]
Same issue: [#4677](https://github.com/ArctosDB/arctos/issues/4677).  Different title.  Still manually deleting them and will use the [other ID bulk-unloader](https://arctos.database.museum/tools/BulkDeleteOtherId.cfm) but that still takes time and is one more task to complete before uploaded records are really complete.

**Comment by campmlc** [comment]
Yes, is it possible for there to be an automated process to delete these
once the associated data in the component loaders has uploaded?

**Comment by Jegelewicz** [comment]
> is it possible for there to be an automated process to delete these once the associated data in the component loaders has uploaded?

This seems like a possibility? When setting records in a component loader to autoload, maybe also offer and "autoload and remove UUID" option? or at least "autoload and create UUID unload file"?
